utility/src/main/java/org/oppia/android/util/enumfilter/BUILD.bazel

```
@@ -0,0 +1,19 @@
+"""
+General purposes utilities for filtering enums.
+"""
+
+
+load("@io_bazel_rules_kotlin//kotlin:android.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "enum_filter_util",
+    srcs = [
+        "EnumFilterUtil.kt",
+    ],
+    visibility = [
+        "//:oppia_api_visibility",
+    ],
```

```suggestion
   visibility = ["//:oppia_api_visibility"],
```

This should fix the failing buildifier lint check.